client/state/imports/site-importer/action-types.js

```diff
@@ -0,0 +1,14 @@
+export const SITE_IMPORTER_IMPORT_START = 'SITE_IMPORTER_IMPORT_START';
```

Comment:
Note: These action-types will be moved out in to the proper `state/action-types.js` before merging / once the logic is all agreed upon.